styles/stripes.h

```diff
@@ -62,31 +90,47 @@ class StripesBase {
 public:
   void run(BladeBase* base, int width, int speed) {
     colors_.run(base);
-    
+
     uint32_t now_micros = micros();
     int32_t delta_micros = now_micros - last_micros_;
     last_micros_ = now_micros;
 
-    m = MOD(m + delta_micros * speed / 333, colors_.size * 341*1024);
-    mult_ = (50000*1024 / width);
+    m = MOD(m + delta_micros * speed / 333, colors_.size * 341 * 1024);
+    mult_ = (50000 * 1024 / width);
   }
   SimpleColor getColor(int led) {
     // p = 0..341*len(colors)
     int p = ((m + led * mult_) >> 10) % (colors_.size * 341);
-    
+
     SimpleColor ret;
-    ret.c = Color16(0,0,0);
+    ret.c = Color16(0, 0, 0);
     colors_.get(led, p, &ret);
     colors_.get(led, p + 341 * colors_.size, &ret);
     return ret;
   }
-private:
+protected:
   StripesHelper<COLORS...> colors_;
   uint32_t mult_;
   uint32_t last_micros_;
   int32_t m;
 };
 
+template<class... COLORS>
+class HardStripesBase : public StripesBase<COLORS...> {
+public:
+  using StripesBase<COLORS...>::StripesBase;
```

Comment:
What is this "using" statement for?

Reply:
i think remnants from early on getting inheritance going.
better now?